IpmiFeaturePkg/Library/IpmiBaseLibSmm/IpmiBaseLibSmm.c

```diff
@@ -130,14 +63,13 @@ IpmiSubmitCommand (
 }
 
 /**
-  Routine to send commands to BMC.
+  Gets the current status of the BMC from the IPMI module.
 
-  @param BmcStatus                   The ConnectAllComplete EFI Event.
-  @param ComAddress                  Event context pass to create function
-
-  @retval EFI_SUCCESS:               Get successfully
-  @retval EFI_NOT_AVAILABLE_YET
+  @param[out]   BmcStatus     The current status of the BMC.
+  @param[out]   ComAddress    The address of the BMC.
 
+  @retval   EFI_SUCCESS             Successfully retrieved BMC status
+  @retval   EFI_NOT_AVAILABLE_YET   Ipmi interface is not installed yet.

```

Comment:
nit: I think `EFI_NOT_AVAILABLE_YET` does not seem to be the error code returned from `EFI_LOCATE_PROTOCOL` function type? (only not found, invalid parameter and success are defined)